kotlin-analysis-api/build.gradle.kts

```diff
@@ -57,6 +57,25 @@ dependencies {
 
     implementation(project(":api"))
     implementation(project(":common-util"))
+
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.8.2")
+    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.8.2")
+    testRuntimeOnly("org.junit.jupiter:junit-jupiter-params:5.8.2")
+    testRuntimeOnly("org.junit.platform:junit-platform-suite:1.8.2")
+
+    testImplementation(project(":test-utils"))
+    testImplementation(project(":compiler-plugin"))
```

Comment:
what's the dependency on `compiler-plugin`? it looks that you've extracted the test processors into the `test-utils` module?

Reply:
A few tests depend on internal functions to dump the lookup info.